contents/feedback/empty-status/simple/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Plus } from "@yamada-ui/lucide"
+import { Text, Button, Flex, Wrap, Icon } from "@yamada-ui/react"
+import type { FC } from "react"
+import { FaRegCreditCard } from "react-icons/fa6"
```

Comment:
If possible, please use only `@yamada-ui/lucide`.

Reply:
I couldn't find an icon that I really liked in `@yamada-ui/lucide`, so I had to use `react-icons`. 😢

Reply:
You can create a new file called `credit-card-icon.tsx` and have a credit card SVG, instead of adding a new library to package.json.